ci-operator/config/stolostron/grafana/stolostron-grafana-global-hub-release-2.9.yaml

```diff
@@ -0,0 +1,100 @@
+base_images:
+  base:
+    name: "4.7"
+    namespace: ocp
+    tag: base
+  stolostron_builder_go1.18-linux:
+    name: builder
+    namespace: stolostron
+    tag: go1.18-linux
+build_root:
+  image_stream_tag:
+    name: builder
+    namespace: stolostron
+    tag: go1.18-linux
```

Comment:
we are using go1.19-linux now for grafana 9.5.6.

Reply:
updated to golang 1.19